build/zip.py

```diff
@@ -68,6 +68,20 @@ def main(args):
         _zip_dir(path, zip_file, archive_name)
       else:
         zip_file.write(path, archive_name)
+  if args.entitlements or args.without_entitlements:
+    entitlement_txt = open('/tmp/entitlements.txt', 'w+')
```

Comment:
is it safe to hardcode this temp path here? what about instead of writing to disk, and then copying to the zip, writing from memory directly to zip with https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.writestr

Reply:
True, working on it